ansible_ai_connect/users/authz_checker.py

```diff
@@ -77,6 +120,15 @@ def fatal_exception(exc) -> bool:
         return False
 
 
+def log_backoff_exception(details):
+    _, exc, _ = sys.exc_info()
+    logger.info(str(exc))
+    logger.info(
```

Comment:
should this be using `.error()`?

Reply:
@jameswnl  thanks for your comment.  Since these are for the exceptions that trigger retries, I think we can use `.info()` here.

However, it is possible that we might be retrying after receiving more critical exceptions. Until now we did not log them and were not sure what kind exceptions were thrown... If those un-retryable exceptions are identified, we should not retry API call and handle those exceptions separately. What do you think?